content/docs/conditional-rendering.md

```diff
@@ -42,15 +42,15 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
+[**Coba di Codepen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+Contoh di atas merender komponen greeting yang berbeda berdasarkan nilai prop `isLoggedIn`.
```

Comment:
@resir014, is `merender` and `prop` valid?

Reply:
The majority of pages uses `me-*render*`, so I guess we should go with that.There's only one page that uses `merender`, so we might update that to make it consistent.

*Prop* should be italicised, e.g. `*prop*`